mod-record-specifications-validator/src/main/java/org/folio/rspec/validation/converter/marc/Marc4jConverter.java

```diff
@@ -84,7 +84,8 @@ private List<MarcIndicator> convertIndicators(DataField field, Reference referen
 
   private MarcIndicator toMarcIndicator(Reference fieldReference, char indicatorValue, int indicatorIndex) {
     var reference = Reference.forIndicator(fieldReference, indicatorIndex);
-    return new MarcIndicator(reference, indicatorValue);
+    var value = indicatorValue == 32 ? '#' : indicatorValue;
```

Comment:
would be nice to extract to some constant with meaningful name than hard coding values of ```32```

Reply:
Right, I've refactored it. Thanks